meta/assessment/originalDataPoint/originalDataPoint.ts

```diff
@@ -1,24 +1,24 @@
 import { Arrays } from '@core/utils'
 
 import { ODPDataSourceMethod } from './odpDataSource'
-import { ODPEditStatus } from './odpEditStatus'
+// import { ODPEditStatus } from './odpEditStatus'
 import { ODPNationalClass } from './odpNationalClass'
-import { ODPValidation } from './odpValidation'
+// import { ODPValidation } from './odpValidation'
 
 export const ODPYears: Array<number> = Arrays.reverse(Arrays.range(1950, 2021))
 
 export interface OriginalDataPoint {
-  countryIso?: string
+  id: number
+  countryIso: string
```

Comment:
this is optional

Reply:
<img width="669" alt="Schermata 2022-02-14 alle 20 59 05" src="https://user-images.githubusercontent.com/10047945/153937217-43fe8bd0-301e-4f2c-b425-8b7a7df835ec.png">